src/library_webgl.js

```diff
@@ -15,6 +15,51 @@ var LibraryGL = {
   _tempFixedLengthArray__postset: 'for (var i = 0; i < 32; i++) __tempFixedLengthArray.push(new Array(i));',
   _tempFixedLengthArray: [],
 
+  _heapObjectForWebGLType: function(type) {
+    type -= 0x1400;
```

Comment:
please add a comment that the subtraction is for shrinking the other constants later for JS size (I assume that's the reason? not immediately obvious to the reader I think)